templates/tour.html

```diff
@@ -11,6 +11,51 @@ <h1 class="u-no-margin--bottom" itemprop="description">Overview and new features
     </div>
 </section>
 
+<section class="p-strip is-deep is-bordered" id="micro-clouds">
+    <div class="row">
+        <div class="col-12">
+            <h2>Create custom VMs with KVM Micro-clouds</h2>
+        </div>
+        <div class="u-equal-height p-divider">
+            <div class="col-4 p-divider__block">
+                <div class="u-sv2">
+                    <a class="u-align--center venobox venobox--expand vbox-item" href="{{ ASSET_SERVER_URL }}952c8938-overview-of-kvms.png" data-gall="tourGallery" title="Create custom VMs with KVM Micro-clouds" aria-label="Modal link to view larger screenshot of create custom VMs with KVM Micro-clouds">
+                      <img class="p-image--bordered" src="{{ ASSET_SERVER_URL }}952c8938-overview-of-kvms.png?h=172" alt="Screenshot of create custom VMs with KVM Micro-clouds"  height="192" />
+                    </a>
+                </div>
+                <h3>Graphical overview of your KVM host’s resources</h3>
+                <p>MAAS shows your KVM pod’s CPU cores and RAM, the free space in storage pools.</p>
+                <p class="u-sv2">Manage and visualise overcommit ratios.</p>
+                <p><a href="https://docs.maas.io/2.5/en/manage-pods-webui#add-a-kvm-host"
+                      onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'KVM' });"
+                      class="p-link--external"
+                      aria-label="External link to KVM in MAAS">Learn more about KVM in MAAS</a>
+                </p>
+            </div>
+            <div class="col-4 p-divider__block">
+                <div class="u-sv2">
+                    <a href="{{ ASSET_SERVER_URL }}28b7223a-choose-storage-from-multiple-pools.png" class="u-align--center venobox venobox--expand" data-gall="tourGallery" title="Choose storage from multiple pools" aria-label="Modal link to view larger screenshot of Choose storage from multiple pools">
+                        <img src="{{ ASSET_SERVER_URL }}28b7223a-choose-storage-from-multiple-pools.png?h=172" class="p-image--bordered" height="192" alt="Screenshot of the subnets listing" />
+                    </a>
+                </div>
+                <h3>Choose storage from multiple pools</h3>
+                <p>Assign one more volumes from the default and other available pools.</p>
```

Comment:
@cassiocassio This sentence reads really weird. Is it supposed to be `Assign one or more volumes` or `Assign more volumes`?

Reply:
good catch :-) 
```suggestion
                <p>Assign one or more more volumes, from the available storage pools.</p>
```